File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		while (true) {
			int A = sc.nextInt();
			int B = sc.nextInt();
			int C = sc.nextInt();

			if (A == 0 || B == 0 || C == 0)
				break;
			else {
				String output = "";
				int maxV = Math.max(A, B);
				maxV = Math.max(maxV, C);

				if (maxV >= A + B || maxV >= B + C || maxV >= C + A)
					output = "Invalid";
				else if (A == B && B == C && C == A)
					output = "Equilateral";
				else if (A == B || B == C || C == A)
					output = "Isosceles";
				else
					output = "Scalene";

				System.out.println(output);
			}
		}
		sc.close();
	}
}

File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.py
while True:
    A, B, C = map(int, input().split())
    if A == 0 or B == 0 or C == 0:
        break
    else:
        output = ""
        maxV = max(A, B, C)
        if maxV >= A + B or maxV >= B + C or maxV >= C + A:
            output = "Invalid"
        elif A == B == C:
            output = "Equilateral"
        elif A == B or B == C or C == A:
            output = "Isosceles"
        else:
            output = "Scalene"
        print(output)